client/style.sass

```diff
@@ -49,19 +55,43 @@ body
 	&:hover
 		cursor: pointer
 
-.MapContainer
+.mapContainer
 	@include library.rounded-border
 	display: flex
-	justify-content: center
+	width: 100%
 	height: 200px
 	background-color: gray
 	margin: 3px 0
 
-.TopBar
+.topBar
+	display: flex
+	flex-direction: row
+	justify-content: space-between
 	background-color: library.$header-colour
 	height: 100px
 
-.test
-	background-color: purple
-	height: 200px
-	width: 50px
+.floatingButtons
+	display: flex
+	justify-content: flex-end
+	position: fixed
+	bottom: 0
+	right: 0
+	z-index: 99
+	width: 200px
+
+.floatingContainer
+	display: flex
+	flex-direction: column
+
+.floatingLogo
+	height: 60px
+	width: 60px
+	margin: 0 5px 5px 0
+	border-radius: 25px
+	box-shadow: 0 0 8px #a9a9a9
+	&:hover
+		cursor: pointer
+
+@media (screen and max-width: library.$max-width)
+	.floatingButtons
+		display: none
```

Comment:
More a comprehension comment - this is for anything that is a screen and has a width less than our specified max width (925px). When the max screen width > 925px, `.floatingButtons` has a normal display. As in previous comment, where does the 925px width come from, convention for a large phone?

Reply:
If the screen is smaller than 925, you actually cover up the pagination forward button, so I just decided to remove them, rather than do more page rebuilding